src/flask_debugtoolbar/panels/profiler.py

```diff
@@ -88,7 +92,14 @@ def process_response(self, request, response):
 
             self.stats = stats
             self.function_calls = function_calls
-            # destroy the profiler just in case
```

Comment:
Hmm, this comment seems semi-interesting, although I don't see an explicit `del` call here. Does it somehow jus t fall out of scope?

Just wondering if this is a contextual comment we should continue pulling forward in the codebase or if it's outdated.

Reply:
I thought it's outdated but if you think otherwise I can restore it.